lib/datadog/tracing/distributed/metadata/b3.rb

```diff
@@ -1,55 +0,0 @@
-# typed: true
-
-require_relative '../helpers'
-require_relative 'parser'
-require_relative '../headers/ext'
-require_relative '../../trace_digest'
-
-module Datadog
-  module Tracing
-    module Distributed
-      module Metadata
-        # B3 provides helpers to inject or extract metadata for B3 style headers
-        module B3
```

Comment:
Whaaaaat this entire class was copy-pasted before mostly to change headers to metadata :O :O :O 

Big thanks for improving this! 🙇

Reply:
That's the main reason I'm doing this: I have to add a new propagator, and didn't want to write 2.